tests/test_common.py

```diff
@@ -0,0 +1,40 @@
+import pathlib
+
+import pytest
+from rdflib import Graph
+
+from pytravharv.common import insert_resource_into_graph
+
+INPUT_PATH = pathlib.Path(__file__).parent / "inputs"
+
+
+@pytest.fixture
+def test_graph():
+    return Graph()
```

Comment:
seriously?

consider more useful fixture or builder methods, and move them into conftest.py
see e.g. https://github.com/vliz-be-opsci/py-RDF-store/blob/main/tests/conftest.py#L78